src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -64,18 +53,9 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
   window['Blazor'].reconnect = reconnect;
 
   const reconnectTask = reconnect(initialConnection);
```

Comment:
Should we be calling reconnect on the initial connection?

Reply:
It does.

Reply:
If we have an existing connection it will attempt to reuse it instead of creating a new one. This was here before my changes 👍

Reply:
It seems really suspect. Reading the code, it looks like it'll call `StartCircuit` followed by a `ConnectCircuit` on the server and the server would think the client went from connected -> disconnected -> connected without knowing about it. You should be able to tell if you turn up the logs in the test app up to `Debug`. While not actively harmful, it seems like incorrect behavior to have by default. Could you quickly confirm?

Reply:
We tried this out and pranav was SUPER CORRECT. this can be removed too now because there's no need to to ever call `ConnectCircuit` and `StartCircuit` for the same connection.